code/modules/organs/subtypes/parasite.dm

```diff
@@ -199,3 +201,59 @@
 		target << "<span class='warning'>Your mind suddenly grows dark as the unity of the Hive is torn from you.</span>"
 	removed_langs = 0
 	..()
+
+/obj/item/organ/parasite/zombie
+	name = "black tumor"
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "blacktumor"
+	dead_icon = "blacktumor"
+
+	organ_tag = "zombie"
+
+	parent_organ = "chest"
+	stage_interval = 150
+
+/obj/item/organ/parasite/zombie/process()
+	..()
+
+	if (!owner)
+		return
+
+	if(prob(10) && !(owner.species.flags & NO_PAIN))
+		owner << "<span class='warning'>You feel a burning sensation on your skin!</span>"
+		owner.make_jittery(10)
+
+	else if(prob(10))
+		owner.emote("moan")
+
+	if(stage >= 2)
+		if(prob(15))
+			owner.emote("scream")
+			if(!isundead(owner))
+				owner.adjustBrainLoss(2, 55)
+
+		else if(prob(10))
+			if(!isundead(owner))
+				owner << "<span class='warning'>You feel sick.</span>"
+				owner.adjustToxLoss(5)
+				owner.delayed_vomit()
+
+	if(stage >= 3)
+		if(prob(10))
+			if(isundead(owner))
+				owner.adjustBruteLoss(-30)
+				owner.adjustFireLoss(-30)
+			else
+				owner << "<span class='cult'>You feel an insatiable hunger.</span>"
+				owner.nutrition = -1
+
+	if(stage >= 4)
```

Comment:
Real quick. Are these stages supposed to stack, or is only one supposed to be active at any one time? At present, they stack. Ergo, someone with stage 4 or 5 will also have the effects from stages 2 - 3.

Reply:
Yes, they are supossed to stack, in the same manner as the k'ois parasite does.